features/step_definitions/basic_steps.rb

```diff
@@ -59,18 +59,18 @@
 Then /^I should( not)? see an edit button for "(.*?)" charity$/ do |negate, name|
   expectation_method = negate ? :not_to : :to
   org = Organisation.find_by_name name
-  expect(page).send(expectation_method, have_link('Edit', :href => edit_organisation_path(org.id)))
+  expect(page).send(expectation_method, have_link('Edit', href: edit_organisation_path(org.id)))
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)